File: src/targets/ruby/native/fixtures/multipart-form-data-no-params.rb
require 'uri'
require 'net/http'
<<<<<<< HEAD
require 'openssl'

url = URI("https://httpbin.org/anything")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true
=======

url = URI("http://mockbin.com/har")

http = Net::HTTP.new(url.host, url.port)
>>>>>>> upstream/master

request = Net::HTTP::Post.new(url)
request["Content-Type"] = 'multipart/form-data'

response = http.request(request)
puts response.read_body

File: src/targets/go/native/fixtures/multipart-form-data-no-params.go
package main

import (
	"fmt"
	"net/http"
<<<<<<< HEAD
	"io/ioutil"
=======
	"io"
>>>>>>> upstream/master
)

func main() {

<<<<<<< HEAD
	url := "https://httpbin.org/anything"
=======
	url := "http://mockbin.com/har"
>>>>>>> upstream/master

	req, _ := http.NewRequest("POST", url, nil)

	req.Header.Add("Content-Type", "multipart/form-data")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
<<<<<<< HEAD
	body, _ := ioutil.ReadAll(res.Body)
=======
	body, _ := io.ReadAll(res.Body)
>>>>>>> upstream/master

	fmt.Println(res)
	fmt.Println(string(body))

}